src/OpenTelemetry.Instrumentation.AspNet/README.md

```diff
@@ -1,64 +1,159 @@
-# ASP.NET Instrumentation for OpenTelemetry .NET
+# ASP.NET Instrumentation for OpenTelemetry
 
 [![NuGet](https://img.shields.io/nuget/v/OpenTelemetry.Instrumentation.AspNet.svg)](https://www.nuget.org/packages/OpenTelemetry.Instrumentation.AspNet)
 [![NuGet](https://img.shields.io/nuget/dt/OpenTelemetry.Instrumentation.AspNet.svg)](https://www.nuget.org/packages/OpenTelemetry.Instrumentation.AspNet)
 
 Automatically instruments the incoming requests to
 [ASP.NET](https://docs.microsoft.com/aspnet/overview).
 
-## Installation
+## Steps to enable OpenTelemetry.Instrumentation.AspNet
```

Comment:
I would suggest that we keep the top level hierarchy consistent with the other README files:

```markdown
## Installation
## Configuration
```

Reply:
mm.. Not sure what is the best way to organize this..
Most package needs installation. But some package like the API/SDK are implicitly brought by other package in many cases.

This PR organizes the doc based on the actual steps one needs to follow. I want to get opinions for this approach. I intend to convert all other docs (instrumentation to begin with) to this format.
Some additional points to consider:
1. Installing a package is not enough to use it.. It must be followed by some code change/config change. Because of this I prefer to keep "install package" as part of steps required to use/enable this package.

2. Though most repos do it, I wonder do we have to explicitly tell "how to install" a package? We can assume users know how to install any .net package in general, and this is not different. So we can simply state "Install this [package name](nugetlink)"

Reply:
Good points.